src/screens/Products/index.js

```diff
@@ -19,16 +19,11 @@ function mapDispatchToProps(dispatch) {
 //We all need to add our own connect methods like it has been done under screens/ArtisanHub/index.js
 
 import { default as UnconnectedAddProduct } from './AddProduct'
+import { default as UnconnectedProductList } from './ProductList'
 import { default as UnconnedtedProductDetail } from './ProductDetail'
 
-const AddProduct = connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(UnconnectedAddProduct)
+const AddProduct = connect(mapStateToProps, mapDispatchToProps)(UnconnectedAddProduct)
```

Comment:
Also good job and changing this since we decided to keep imports on one line until needs to be wrapped.